lib/vsc/utils/timestamp.py

```diff
@@ -132,10 +132,10 @@ def write_timestamp(filename, timestamp):
     cache.close()
 
 
-def retrieve_timestamp_with_default(filename, start_timestamp=None, default_timestamp="2014010100000Z", delta=-10):
+def retrieve_timestamp_with_default(filename, start_timestamp=None, default_timestamp="201401010000", delta=-10):
```

Comment:
i had this also in the sync class in accoutnpage-clients repo. didn't know it was defined heere too
i prefer to keep the zulu format (but it's missing `00` from seconds)
default should be a constant so you can unittest it properly